tests/unit/lms/services/lti_registration_test.py

```diff
@@ -10,6 +10,9 @@ class TestLTIRegistrationService:
     def test_get(self, svc, registration):
         assert svc.get(registration.issuer, registration.client_id) == registration
 
+    def test_without_client_id(self, svc, registration):
+        assert svc.get(registration.issuer) == registration
```

Comment:
~Does this fail if more than one matches? Should it?~ Should we test for it?

Yes. `one_or_none` should do this for us.